AstSemantics.md

```diff
@@ -531,3 +531,12 @@ round-to-nearest ties-to-even rounding.
 Truncation from floating point to integer where IEEE 754-2008 would specify an
 invalid operation exception (e.g. when the floating point value is NaN or
 outside the range which rounds to an integer in range) traps.
+
+## Feature test
+
+To support [feature testing](FeatureTest.md), an AST node would be provided:
+
+  * `has_feature`: return whether the given feature is supported, identified by string
```

Comment:
I get being always the same within a given instance. What's the motivation for requiring different instances whose lifetime overlap to be the same? Clearly instances on different computers at the same time won't require their implementations to communicate, so where would we draw the line?